python/vtool/util_file.py

```diff
@@ -2568,6 +2568,18 @@ def create_file(name, directory=None, make_unique=False):
 
     return full_path
 
+def remove(name, directory = None):
+    """
+    Remove a file or folder
+    """
+    # using try except because its faster than checking if its a file or folder
+    full_path = delete_file(name, directory)
+    try:
+        shutil.rmtree(full_path)
+    except:
```

Comment:
Should really be avoiding naked exceptions or at least pass the error message. You spent the time implementing a logging system, having it handle exceptions and print out what went wrong is what it's for.

Reply:
I will come back to this.  I need to improve my logging to easily be able to set the amount feedback.